cypress/integration/e2e/management-tab/edit-worker.spec.ts

```diff
@@ -51,4 +52,25 @@ describe("Tab management", () => {
       cy.get('[data-cy="input-civil-time"] input').clear({ force: true }).type("123");
     });
   });
+
+  const testWorker = "Pielęgniarka 2";
+  describe("Editing workers time", () => {
```

Comment:
Add test when workers time is changed from lower value to higher i.e. 1/2 -> 1/1 or add test going from custom workers time (`inne` tab)

Reply:
Done.